runbot_docker_orchest/tests/test_runbot_build.py

```diff
@@ -0,0 +1,104 @@
+from odoo.tests.common import TransactionCase
+from odoo.tools.misc import mute_logger
+import xmlrpc.client
+import logging
+import time
+import unittest
+import os
+
+_logger = logging.getLogger(__name__)
+
+DOCKER_LOGIN = (
```

Comment:
Read the config file `~/.docker/config.json` and check if is defined ['auths']['quay.io']